packages/mdx-deck-code-surfer/src/deck-components.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Notes } from "mdx-deck";
+import { Notes } from "@mdx-deck/components";
```

Comment:
The `mdx-deck` package actually exports all of the components in `@mdx-deck/components`. See [here]( https://github.com/jxnblk/mdx-deck/blob/48c734ffbf082501588d34916228b6c881309d5d/packages/mdx-deck/index.js#L1).